features/nvstore/mbed_lib.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "nvstore",
+	"config": {
+		"num_keys": {
+	        "macro_name": "NVSTORE_MAX_KEYS",
+	        "value": 16,
+	        "help": "Maximal number of allowed NVStore keys"
+	    },
+	    "area_1_address": {
+	        "macro_name": "NVSTORE_AREA_1_ADDRESS",
+	        "help": "Area 1 address"
+	    },
+	    "area_1_size": {
+	        "macro_name": "NVSTORE_AREA_1_SIZE",
+	        "help": "Area 1 size"
+	    },
+	    "area_2_address": {
+	        "macro_name": "NVSTORE_AREA_2_ADDRESS",
+	        "help": "Area 2 address"
+	    },
+	    "area_2_size": {
+	        "macro_name": "NVSTORE_AREA_2_SIZE",
+	        "help": "Area 2 size"
+	    }
+    },
+	"target_overrides": {
+		"K64F": {
```

Comment:
If a target needs to define these, and we got only 2 targets available for this feature, this PR will need to be redirected to own feature branch

Reply:
@sg- @bulislaw @theotherjimmy We need to enhance the definition for additional targets but I think we can do it internally and do not need partners for it. We need some help with targets that we do not have. Also need the new tools from @theotherjimmy

Reply:
@dannybenor Which new tools are these? Is there an associated PR with them, or are they in a different repo?

Reply:
@cmonr This Jira was created to follow up the tools improvements https://jira.arm.com/browse/IOTMORF-1991
For managed BL mode, the addresses are outputs.

Reply:
@dannybenor Sounds good, but I think @0xc0170's question is still unanswered. At this time, this feature is still only targeting a few targets, instead of the entire ecosystem.

Reply:
Going to add support for a whole lotta boards in this PR, hopefully soon.